pytest_fixtures/component/rh_cloud.py

```diff
@@ -35,13 +33,6 @@ def organization_ak_setup(rhcloud_sat_host, rhcloud_manifest_org):
         purpose_role='test-role',
         auto_attach=True,
     ).create()
-    subscription = rhcloud_sat_host.api.Subscription(organization=rhcloud_manifest_org)
```

Comment:
Is there a chance the organization could be in a non-sca mode? If yes, I suggest making this block conditional, someth like `if rhcloud_sat_host.api.Organization(id=rhcloud_manifest_org.id).read().simple_content_access is False:`

Reply:
@pondrejk Test is using `module_entitlement_manifest` fixture, it should return manifest in entitlement mode but I am getting manifest in sca mode.😅

```
def test_manifester(module_org, module_target_sat, module_entitlement_manifest):
    """fesd"""
    module_target_sat.upload_manifest(module_org.id, module_entitlement_manifest.content)
```

Reply:
Need to do further debugging.

Reply:
Will look into it later, added your suggestions. Most of the tests are now passing.